clowdapp.yaml

```diff
@@ -151,13 +151,13 @@ objects:
               httpGet:
                 path: /mgmt/v0/status
                 port: 8000
-              initialDelaySeconds: 10
+              initialDelaySeconds: 120
               periodSeconds: 10
             readinessProbe:
               httpGet:
                 path: /mgmt/v0/status
                 port: 8000
-              initialDelaySeconds: 10
```

Comment:
I'd keep this one as is for readiness, otherwise it will slowdown production rolling deployments.